ZipBuilder/Sources/ZipBuilder/ZipBuilder.swift

```diff
@@ -160,8 +160,8 @@ struct ZipBuilder {
 
     // Break the `inputPods` into a variable since it's helpful when debugging builds to just
     // install a subset of pods, like the following line:
-    //    let inputPods: [String] = ["", "Core", "Analytics", "Storage"]
-    let inputPods = CocoaPod.allCases.map { $0.rawValue }
+    let inputPods: [String] = ["", "Core", "Analytics", "Storage"]
+    //let inputPods = CocoaPod.allCases.map { $0.rawValue }
```

Comment:
Whoops, probably want to take this back :)

Reply:
Done :)